### DISCUSSION TITLE
Práctica Abogabot launch x

### DISCUSSION THREAD

**Discussion by Klyuvert** [Launch-X-Latam/MisionFrontEnd#421] [Show and tell]
No cuento con mucho tiempo pero mejor tarde que nunca. https://github.com/KlyuBM/Abogabot

**Comment by GonDev1** [comment]
Quedo muy bien organizado, le metiste documentación y eso lo hace ver muy pro, tal vez mejoraría la parte intuitiva aprovechando al máximo las herramientas que utilizaste, saludos ✌

**Reply by Klyuvert** [reply]
gracias por el comentario amigo
